mantidproject/mantid
# Polaris Focus Made multiple scattering an optional flag

**Description of work.**

The multiple_scattering flag was required if absorption_corrections was true, despite multiple_scattering not being available.
This changes multiple scattering to be optional.

**To test:**

Download these [files](https://github.com/mantidproject/mantid/files/2357307/test.zip) and unzip them, ensuring that the paths in polaris_config.yaml are set and do_absorb_corrections is True. then run
```
from isis_powder import Polaris, SampleDetails

cylinder_height = 3.0
cylinder_radius = 2.0
cylinder_position = [0.0, 0.0, 0.2]
sample_obj = SampleDetails(height=cylinder_height, radius=cylinder_radius,
                                            center=cylinder_position, shape="cylinder")
                           
sample_obj.set_material(chemical_formula="V")

config_file_path = r"Path\to\polaris_config.yaml"
Polaris_example = Polaris(config_file=config_file_path,user_name="test")
Polaris_example.set_sample_details(sample=sample_obj)

Polaris_example.create_vanadium(first_cycle_run_no=98531)
Polaris_example.focus(input_mode="Individual", run_number="98531-98534")
```
With the path to the config set properly.

it should run to completion and not fail at focus due to not having the multiple_scattering flag set.

Fixes #23449 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
